index.bs

```diff
@@ -1285,6 +1295,17 @@ controls the maximum [=map/size=] of a [=trigger spec map=] for an
 the maximum [=string/length=] of an [=attribution trigger=]'s [=attribution trigger/trigger context ID=].
 Its value is 64.
 
+<dfn>Default max event states</dfn> is a positive integer that controls the
+default [=attribution source/max event states=]. Its value is 3.
+
+<dfn>Max length per attribution scope</dfn> is a positive integer that controls the
+maximum [=string/length=] of an attribution scope from [=attribution source=]'s [=attribution source/attribution scopes=] or [=attribution trigger=]'s [=attribution trigger/attribution scopes=].
```

Comment:
Do we need to check this for triggers? For filters, we don't enforce the limit on trigger side.

Reply:
It won't match any sources, and this would be part of the attribution logic. I slightly prefer to align with filters on this behavior. @apasel422 wdyt?

Reply:
I'm fine aligning it with filters.

Reply:
Done.

Reply:
Shall we remove attribution trigger from this definiton?